mysql/const_test.go

```diff
@@ -148,3 +148,15 @@ func (s *testMySQLConstSuite) TestRealAsFloatMode(c *C) {
 	row = result.Rows()[0]
 	c.Assert(row[1], Equals, "float")
 }
+
+func (s *testMySQLConstSuite) TestNoUnsignedSubTractionMode(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	defer s.cleanEnv(c)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("CREATE TABLE t (a BIGINT, b BIGINT UNSIGNED);")
```

Comment:
We don't need to create a table for this test.

And please add more tests like
SELECT CAST(18446744073709551615 as UNSIGNED) - 1;
SELECT 1 - CAST(18446744073709551615 as UNSIGNED);